examples/kubernetesversions/kubernetes1.7.1.json

```diff
@@ -1,36 +0,0 @@
-{

```

Comment:
This is related to our migrating away from user-specified "major.minor.patch" version specifications in favor of "major.minor" only, with `acs-engine` programmatically determining the latest stable patch version on that user-requested version channel.